test/test_db_utils.py

```diff
@@ -68,11 +69,12 @@ def test_google_credentials():
 #Tests the Course Methods
 def create_test_courses(returned_ids, collection, db):
     for i in range(100):
-        info = dbu.Course.create_course('CS61'+str(i),
-            {"id" : i ,
+        info = dbu.Course.create_course({
+             "id" : i,
```

Comment:
Why an `id` and a `course_ok_id` field?

Reply:
I was just hardcoding the mock id. I suppose I could remove it.